### PR TITLE
add jira update files for platform and access control service

### PR DESCRIPTION
**Pull request by siraj-io**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/39349)

### CODE REVIEW

File: platform-service/release/release-branch-create-pls-versions.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2022 Harness Inc. All rights reserved.
+# Use of this source code is governed by the PolyForm Free Trial 1.0.0 license
+# that can be found in the licenses directory at the root of this repository, also available at
+# https://polyformproject.org/wp-content/uploads/2020/05/PolyForm-Free-Trial-1.0.0.txt.
+set -ex
+
+export FIX_PLS_VERSION="PlatformService_""$VERSION"
+RELDATE=$(date +'%Y-%m-%d')
+IFS="-" read -ra PROJNUM <<< "$KEY"
```
Comment (nataraja2018) [inline platform-service/release/release-branch-create-pls-versions.sh:10]: Remove which is not needed

File: platform-service/release/release-branch-create-pls-versions.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+PROJ="PL"
+response=$(curl -X POST https://harness.atlassian.net/rest/api/2/version/ --write-out '%{http_code}' --output /dev/null --silent --user ${JIRA_USERNAME}:${JIRA_PASSWORD} -H "Content-Type: application/json" -d '{
+  "project":"'$PROJ'",
+  "name": "'"$FIX_PLS_VERSION"'",
+  "releaseDate": "'"$RELDATE"'",
+  "archived": false,
+  "released": true }')
+
+
```
Comment (nataraja2018) [inline platform-service/release/release-branch-create-pls-versions.sh:19]: Fail if fix version is not created.

File: platform-service/release/release-branch-update-jiras.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -ex
+KEYS=$(git log --pretty=oneline --abbrev-commit |\
+      awk "/${PREVIOUS_CUT_COMMIT_MESSAGE}/ {exit} {print}" |\
```
Comment (nataraja2018) [inline platform-service/release/release-branch-update-jiras.sh:5]: Export the PREVIOUS_CUT_COMMIT_MESSAGE from CI Job

**Review by nataraja2018 — CHANGES_REQUESTED** [review]
Please looks into the comments

**Review by siraj-io — COMMENTED** [review]
Review comments implemented

**Pull request merged**